Makefile

```diff
@@ -250,6 +250,14 @@ cross:
 $(notdir $(abspath $(wildcard cmd/*/))): generated_files
 	hack/make-rules/build.sh cmd/$@
 
+# Add rules for kube-scheduler
+#
+# Example:
+#   make kube-scheduler
+.PHONY: kube-scheduler
+kube-scheduler: generated_files
+	hack/make-rules/build.sh plugin/cmd/kube-scheduler
```

Comment:
Can you follow the `$(notdir $(abspath $(wildcard` pattern for `plugin/cmd/*/` here?